docs/config.md

```diff
@@ -93,6 +93,38 @@ lift-attributes = [
   cluster = 'cluster1'
   geo = 'eu'
 
+# Optional configuration for exporting telemetry outside of adx-mon.
+# Exporters are declared here and referenced by name in each collection source.
```

Comment:
Maybe add that exporters send a copy of the matched metrics to the destination and that the metric is still send to kusto as well?

Reply:
Might also be useful to explain the flow and how/when the various keep/drop rules are applied with the various sources.  For example, the exporters apply these rules after all the source level rules are applied.

Reply:
Updated this comment a bit, and expanded a lot more on it in the Exporters section.